api/metrics/metrics.go

```diff
@@ -36,10 +37,10 @@ type LMMetricIngest struct {
 	client   *http.Client
 	url      string
 	batch    bool
-	interval int
+	interval time.Duration
 }
 
-func NewLMMetricIngest(batch bool, interval int) (*LMMetricIngest, error) {
+func NewLMMetricIngest(ctx context.Context, opts []Option) (*LMMetricIngest, error) {
```

Comment:
`opts` in `NewLMMetricIngest` should be` opts ...Option` instead of `opts []Option`.